spec/models/service_spec.rb

```diff
@@ -1,5 +1,12 @@
 require 'rails_helper'
 
 RSpec.describe Service, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  context 'database associations' do
+    it { is_expected.to have_and_belong_to_many(:events) }
+  end
+
+  context 'model attributes' do
+    it { is_expected.to have_db_column(:name).of_type(:string)
+                            .with_options(limit: 50) }
```

Comment:
Expression at 10, 54 should be on its own line.<br>Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 24) spaces for indenting an expression spanning multiple lines.